ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -128,8 +128,7 @@ private KsqlRestApplication(
       final RootDocument rootDocument,
       final StatusResource statusResource,
       final StreamedQueryResource streamedQueryResource,
-      final KsqlResource ksqlResource,
-      final VersionCheckerAgent versionCheckerAgent
+      final KsqlResource ksqlResource
```

Comment:
why have we dropped the version checker agent from the constructor? 

This means we have two instances of the version checker running. One created in `buildApplication` and created in this constructor....